fcm_django/models.py

```diff
@@ -15,7 +15,7 @@ class Device(models.Model):
 		verbose_name=_("Is active"), default=True,
 		help_text=_("Inactive devices will not be sent notifications")
 	)
-	user = models.ForeignKey(SETTINGS["USER_MODEL"], blank=True, null=True)
+	user = models.ForeignKey(SETTINGS["USER_MODEL"], related_name='devices', blank=True, null=True)
```

Comment:
The convention I have seen and used elsewhere in the code is to called the related_name the same as the model. Makes it a bit easier to track magic code. So 'fcm_devices'.

Reply:
Done